src/test/regress/expected/analyze.out

```diff
@@ -0,0 +1,38 @@
+/*
+Testing NULL counts 
+*/
+-- 1. empty table should have NO stats
+create table t (c int) distributed by (c);
```

Comment:
I would use a more verbose table name like 'analyze_test' since there may be other tests running at the same time using 't' as a table name.